src/resources/font.js

```diff
@@ -72,6 +72,17 @@ Object.assign(pc, function () {
 
         open: function (url, data, asset) {
             var font;
+            // convert v2 to v3 font data
```

Comment:
We might actually need to add a v1->v2 conversion here as well. v1 format is missing the `data.info.maps` section and the `map` property for each character.

Reply:
See Arial.json in the engine examples as a v1 sample